sycl/doc/extensions/SubGroup/SYCL_INTEL_sub_group.asciidoc

```diff
@@ -295,6 +305,13 @@ Yes, this is required by OpenCL devices.  Devices that do not require the work-g
 Yes, the four shuffles in this extension are a defining feature of sub-groups.  Higher-level algorithms (such as those in the +SubGroupAlgorithms+ proposal) may build on them, the same way as higher-level algorithms using work-groups build on work-group local memory.
 --
 
+. What should the sub-group size compatible with all features be called?
++
+--
+*RESOLVED*:
+The name adopted is "core", to convey that it supports all core (non-optional) language features and is an integral part of sub-group support provided by the device.  Other names considered are listed here for posterity: "default", "stable", "fixed", "primary".  With the exception of "primary", these terms are easy to misunderstand (i.e. the "default" size may not be chosen by default, the "stable" size is unrelated to the software release cycle, the "fixed" sub-group size may change between devices or compiler releases).  "core" was selected over "primary" because of its simplicity.
```

Comment:
It's likely we'll need an attribute `map_work_group_to(core|thread|socket)` in the future. Then the usage of "core" will be confusing because it will mean "cpu-core" for the one attribute and "core-feature" for the other. Not sure this is important enough to reopen this discussion. Just checking whether it was considered as a problem.

Reply:
I don't think this was considered.  I at least hadn't considered that the word "core" might be overloaded.  Even if we don't introduce new parameters, there's still a risk that somebody running on the CPU will interpret it as "the sub-group size of the core" rather than "the main sub-group size".

This makes me lean more towards "primary".  @bashbaug?

Reply:
Following offline discussion, we're adopting "primary" instead of "core" to avoid this confusion.